CHANGELOG.md

```diff
@@ -42,6 +42,12 @@ This preview version removes Crashes support for UWP. The App Center backend is
 
 * **[Fix]** The in-memory cache of error reports is now cleared when disabling Crashes.
 
+### App Center Push
+
+#### Xamarin
+
+* **[Fix]** Updated Android support packages to 28.0.0.1 version in Push module.
```

Comment:
Is this a bug fix or just updating a dependency? I just wonder this is the one that we need to specify in the CHANGELOG.

Reply:
Yes we should as this would fix a customer issue.